pulitzer.gemspec

```diff
@@ -32,6 +32,9 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'sqlite3', '~> 1.3'
   s.add_development_dependency 'minitest', '~> 5.1'
   s.add_development_dependency 'byebug', '~> 8.0'
+  s.add_development_dependency 'pry-rails', '~> 0.3', '< 1.0'
```

Comment:
If pry is going away and byebug is the new thing everyone is using, we will eventually want to shift from Pry to ByeBug. But maybe we can wait until we upgrade rails?